tests/test_cmd_icc_profile.sh

```diff
@@ -90,6 +90,12 @@ pushd ${TMP_DIR}
   "${AVIFDEC}" "${ENCODED_FILE}" "${DECODED_FILE}"
   "${IMAGEMAGICK}" "${DECODED_FILE}" -profile "${SRGB_ICC}" "${CORRECTED_FILE}"
   "${ARE_IMAGES_EQUAL}" "${REFERENCE_GRAY_PNG}" "${CORRECTED_FILE}" 0 45
+
+  # Check --cicp flag works
+  "${AVIFENC}" --cicp 9/12/8 -s 8 "${INPUT_COLOR_PNG}" -o "${ENCODED_FILE}"
+  "${AVIFDEC}" "${ENCODED_FILE}" "${DECODED_FILE}" | grep "Color Primaries" | grep 9
```

Comment:
I'm not familiar with the format, so maybe 2 greps are more stable if the structure changes. Otherwise .* could work, but either is fine.

Reply:
Yeah I wanted to make it flexible but I realized that my version would also match e.g. "18" if grepped for "8" so I made the regex a bit more precise.